src/controller/subtitle-stream-controller.js

```diff
@@ -245,6 +263,31 @@ export class SubtitleStreamController extends BaseStreamController {
   }
 
   onMediaSeeking () {
-    this.fragPrevious = null;
+    // if (this.state === State.FRAG_LOADING) {
+    // WIP not reliable as the loader is active in State.IDLE
+
+    if (this.fragCurrent) {
+      const currentTime = this.media ? this.media.currentTime : null;
+      const tolerance = this.config.maxFragLookUpTolerance;
+      const fragStartOffset = this.fragCurrent.start - tolerance;
+      const fragEndOffset = this.fragCurrent.start + this.fragCurrent.duration + tolerance;
+
+      // check if we seek position will be out of currently loaded frag range : if out cancel frag load, if in, don't do anything
```

Comment:
```suggestion
      // check if position will be out of currently loaded frag range after seeking : if out, cancel frag load, if in, don't do anything
```
"check if we seek position" read as a typo to me. Does this suggested change read better to you?

Reply:
yes, typo :upside_down_face: